### PR TITLE
PR for BZ-1199993, BZ-1260881 and 1260909

### PR DESCRIPTION
**Pull request by mrietveld**
See: 

https://bugzilla.redhat.com/show_bug.cgi?id=1199993
https://bugzilla.redhat.com/show_bug.cgi?id=1260881
https://bugzilla.redhat.com/show_bug.cgi?id=1260909

### CODE REVIEW

**Comment by mrietveld** [comment]
jenkins, please test

File: kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/data/QueryResourceQueryParametersTest.java [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+                    param.toLowerCase().equals(param) );
+            assertTrue( "Param \"" + param + "\" is used twice",
+                    params.add(param) );
+        }
+        assertTrue( "Param \"" +  PROC_INST_ID_PARAM_NAME + "\" is used twice",
+                params.add(PROC_INST_ID_PARAM_NAME) );
+    }
+
+    @Test
+    @Ignore
```
Comment (mswiderski) [inline kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/data/QueryResourceQueryParametersTest.java]: is this really a test or some kind of helper method?

File: kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/data/QueryResourceQueryParametersTest.java [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+               break;
+           }
+           System.out.println( "case " + num + ": // " + action.getValue()  + "\nbreak;");
+       }
+    }
+
+
+    @Test
+    @Ignore
+    public void debugPrintQueryParameterIds() throws Exception {
```
Comment (mswiderski) [inline kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/data/QueryResourceQueryParametersTest.java]: same here - is this really a test or some kind of helper method?

**Comment by mswiderski** [comment]
besides these two very very minor comments +1, will just wait for reply about the comments and then merge

File: kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/data/QueryResourceQueryParametersTest.java (hunk 1 above)
Comment (mrietveld) [inline kie-remote/kie-remote-services/src/test/java/org/kie/remote/services/rest/query/data/QueryResourceQueryParametersTest.java]: Good point, I'll remove the `@Test` and `@Ignore`. 

(These are helper methods).

**Comment by mrietveld** [comment]
Branch updated.

**Comment by mrietveld** [comment]
jenkins, please retest.

**Comment by mswiderski** [comment]
merged to master, thanks

**Comment by mrietveld** [comment]
Thank you! :)

**Pull request closed without merging**